lib/koans/19_genservers.ex

```diff
@@ -0,0 +1,141 @@
+defmodule GenServers do
+  use Koans
+
+  @intro "GenServers"
+
+  defmodule BicycleLock do
+    use GenServer
+
+    @init_password 1234
```

Comment:
I'm thinking having a default password is a little distracting. Why not just always require them to specify a password so that usage is explicit in the examples?